server/views/pages/adjustments/remand/edit.njk

```diff
@@ -57,18 +57,16 @@
         isEdit: true
       }) }}
 
-      <h2 class="govuk-heading-m">Now save the remand time</h2>
-
       {% if model.showUnusedMessage %}
         <p class="govuk-body">The updates will change the amount of unused deductions. Check the unused remand alert on NOMIS.</p>
       {% endif %}
 
       <form method="post">
         <input type="hidden" name="_csrf" value="{{ csrfToken }}"/>
         <div class="govuk-button-group">
-          {% if not model.remandRelatedValidationSummary().errorList.length %}
+          {% if not model.remandRelatedValidationSummary().errorList.length and model.isModified() %}
```

Comment:
is it worth adding a unit test for this?

Reply:
I've written 2 unit tests for this